code/web/package.json

```diff
@@ -29,6 +29,7 @@
     "babel-loader": "8.1.0",
     "dotenv-webpack": "^1.7.0",
     "nodemon": "2.0.3",
+    "redux-devtools-extension": "^2.13.8",
```

Comment:
Glad to see this is added in, and I am curious to see the implications of all of the different package.jsons we will be dealing with. 
But this line should be a one and done :)